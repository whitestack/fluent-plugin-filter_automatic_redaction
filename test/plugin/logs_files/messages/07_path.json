[
  {
    "input_message": "Lorem Ipsum /home/john-doe Lorem Ipsum"
  },
  {
    "input_message": "Lorem Ipsum /home/jane-doe Lorem Ipsum"
  }
]