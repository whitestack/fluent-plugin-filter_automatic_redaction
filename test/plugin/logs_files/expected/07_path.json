[
  {
    "input_message": "Lorem Ipsum /home/john-doe Lorem Ipsum",
    "output_message": "Lorem Ipsum <PATH> Lorem Ipsum"
  },
  {
    "input_message": "Lorem Ipsum /home/jane-doe Lorem Ipsum",
    "output_message": "Lorem Ipsum <PATH> Lorem Ipsum"
  }
]